src/utils/types.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable max-len */

```

Comment:
shouldn't be needed? we use prettier so it should be disabled?

Reply:
yeap, removed the eslint rule